### ISSUE TITLE
Doesn't detect syntax error

### ISSUE THREAD

**Issue by slackorama** [rodjek/puppet-lint#187]
Given the following:

```
 file { '/etc/motd',
    content => 'I will never run'
 }
```

This will pass puppet-lint but fail when puppet tries to run it:

```
 % puppet apply bad.pp
Could not parse for environment production: Syntax error at ','; expected '}' at /tmp/vagrant-puppet/manifests/base.pp:1 on node lucid64.

% puppet-lint --version 
Puppet-lint 0.3.2
```
